src/Table/AbstractTable.php

```diff
@@ -263,10 +263,16 @@ public function delete(RowInterface $row)
      * @return RowInterface
      *
      */
-    public function newRow(array $cols = [])
+    public function newRow(array $colsVals = [])
```

Comment:
@pmjones I am @yespire have discussed something regarding the variable representation on Atlas. We have not opened an issue, just for not sure about it. For your consideration it is consistency of `$cols` and `$colVals` in some places. When it comes to `$cols` is it only column names , `$colVals` , column names with values. ( Associative array ) . If you have any inputs / think need to reopen an issue happy to discuss the same.

Thank you.